whitepaper.md

```diff
@@ -0,0 +1,170 @@
+
+1. **Separation of Concerns **- Data is only sent to the relevant parties of a transaction, with an optional hash of that data broadcast to all blockchain nodes. This means of privacy has been implemented by R3’s Corda, JPMorgan’s Quorom, and Hyperledger Fabric through it’s channels model. While this method achieves the privacy and data locality concerns, it has a few drawbacks. The key disadvantage of this model is that it creates a number of ‘sub-chains’ and transferring assets between these chains adds significant complexity or reduces confidentiality.
+
+2. **Computation on Encrypted Data** - In this approach, encrypted data is broadcast across the entire network in a way that the data can be computed on. This may entail advanced cryptographic techniques such as homomorphic encryption or zero-knowledge proofs, or leverage a trusted execution environment such as Intel® Software Guard Extensions (Intel® SGX). This method too comes with several drawbacks. Such drawbacks may include the need for trusted hardware or performance and usability limitations with advanced cryptographic techniques.
+
+Currently Sawtooth does not dictate a privacy and confidentiality approach. The Sawtooth team has experimented with several approaches, including trusted computation of transaction off-chain, which can be found here - [https://sawtooth.hyperledger.org/docs/core/releases/0.8.8/examples/private_utxo/overview.html](https://sawtooth.hyperledger.org/docs/core/releases/0.8.8/examples/private_utxo/overview.html).
+
+This area is an active area of research and a key priority for future Sawtooth releases. The Sawtooth team believes that different privacy models are appropriate for different deployments and intends to leverage data isolation, cryptographic, and trusted hardware techniques in their approach.
+
+**Decentralized Governance**
+
+One of the unique features of Sawtooth is the on-chain governance mechanism enabled by the ‘settings’ transaction family. The settings transaction family provides a methodology for storing on-chain configuration settings such as which consensus mechanism is used and which transaction processors are enabled on the network
+
+This design of the settings transaction family supports two authorization options, either a single authorized key which can make changes or multiple authorized keys. In the case of multiple keys, a percentage of votes signed by the keys is required before a change is made.
+
+**Sawtooth and Ethereum Compatibility**
+
+The primary goal of the Sawtooth-Ethereum integration project, affectionately dubbed "Seth", is to add support for running Ethereum Virtual Machine smart contracts to the Hyperledger Sawtooth platform. In order to make this possible, the Hyperledger Sawtooth project worked with the [Hyperledger Burrow](https://github.com/hyperledger/burrow) project to integrate their EVM implementation, the Burrow EVM, into the Hyperledger Sawtooth platform.
```

Comment:
Change introductory sentence to read as follows, with link to `sawtooth-seth` repository:

> The primary goal of the Sawtooth-Ethereum integration project [Seth](https://github.com/hyperledger/sawtooth-seth/) is to ...